assets/constants.ts

```diff
@@ -5280,28 +5307,37 @@ export default {
       // eslint-disable-next-line max-len
-      "Use respectful ways and formal prefixes to address people. For example, use “سيدي” and “سيدتي”. ",
+      "When writing a number, remember that \",\" in English corresponds to \".\" in Portuguese, and vice-versa.",
       // eslint-disable-next-line max-len
-      "If the name has a meaning in Arabic, or in English, such as Baker or Crumb, always use words that indicate that they are names before writing the name itself. For example, you can use one of the following words depending on the context; “السيد، السيدة، العم، الجد، الجدة، الآنسة.”",
-      "Use the same voice (active or passive) as in the original English Text",
+      "Prefer to use friendly words and sentences for children. For example, instead of using \"Diga-me o nome...\" or \"vou pedir-lhe ajuda\", you can write the sentence with a more informal language like \"Me diga o nome...\" or \"vou pedir a sua ajuda\".",
       // eslint-disable-next-line max-len
-      "Preserve punctuation and bolding. If the original content has bold text, make sure it is bold in Arabic as well. If there are bullet points, double quotes, etc., make sure that the translated content also has bullet points and double quotes.",
+      "In English some nouns/articles are neutral. In Portuguese the nouns are usually gendered. Be careful not to generate inconsistencies and make sure to use the correct article gender for the noun gender. For example: Nina and Sandra loved the cake. They went out to buy more. In Portuguese: Nina e Sandra adoraram o bolo. Elas saíram para comprar mais.",
       // eslint-disable-next-line max-len
-      "Use the hyperlinks to different cards as shown in the original English Text."
+      "Preserve punctuation and bolding. If the original content has bold text, make sure it is bold in Portuguese as well. If there are bullet points, double quotes, etc., make sure that the translated content also has bullet points and double quotes.",
+      // eslint-disable-next-line max-len
+      "If the original card has “components” (such as pictures, links, and equations), these need to be added to the translated content. You can use the “Copy tool” for this -- click on the Copy tool and then click on the component you want to carry over. Also, double-click on the image and translate the alt text (and caption, if any).",
```

Comment:
Seems like a general instruction that may apply to other languages too?

Reply:
There should be a tooltip for showing how to use the copy tool now—this might be just an extra nudge for this language's translators that the writer wanted to give..